src/main/resources/io/vertx/ext/shell/shell.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<title>Vert.x Shell</title>
+
+<h1>Vert.x Shell</h1>
+
+<style>
+
+  @import url(http://fonts.googleapis.com/css?family=Ubuntu:400,500,700,400italic);
```

Comment:
actually it's not a problem, it will fallback on another monospaced font, I tried it works.